src/containers/Home/Home.test.tsx

```diff
@@ -28,7 +28,7 @@ describe("Home", () => {
     userEvent.type(input, "78701");
     expect(input.value).toBe("78701");
 
-    const submit = screen.getByRole("button", { name: /submit/i });
+    const submit = screen.getByRole("button", { name: /Submit/i });
```

Comment:
Minor, but this change isn't doing anything :) The `i` flag on the regex means it should match insensitively.

Reply:
I changed the tests to look for the resource key, then changed them back to look for the English string once I fixed the tests. Just ignore this change it doesn't matter

Reply:
All good, you mentioned in the channel you were looking to improve your js, so I just wanted to mention this.